Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Ride.hs

```diff
@@ -67,6 +68,8 @@ data Ride = Ride
     driverArrivalTime :: Maybe UTCTime,
     rideStartTime :: Maybe UTCTime,
     rideEndTime :: Maybe UTCTime,
+    fromLocationId :: Id TripLocation,
+    toLocationId :: Maybe (Id TripLocation),
```

Comment:
Why do you need that? In 90% places we have booking and ride at the same scope, just fetch this from booking?